tensorflow/python/ops/math_ops.py

```diff
@@ -1136,6 +1136,14 @@ def div_no_nan(x, y, name=None):
 @dispatch.add_dispatch_support
 def multiply_no_nan(x, y, name=None):
   """Computes the product of x and y and returns 0 if the y is zero, even if x is NaN or infinite.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> x = tf.constant([[1.0, 3.0], [5.0, 4.0]])
+    >> y = tf.constant([[7.0, 2.0], [1.0, 9.0]])
+    >> tf.math.multiply_no_nan(x, y) # [[7, 6] [5, 36]]
```

Comment:
You don't need the tf.constants, just pass the python lists of values and let tf convert

Reply:
though actually passing NaN in a few entries and showing the result would be useful